lib/scala/searcher/src/main/scala/org/enso/searcher/sql/SqlDatabase.scala

```diff
@@ -19,23 +19,26 @@ final class SqlDatabase(config: Option[Config] = None)
 
   private var db: SQLiteProfile.backend.Database = _
 
-  open()
-
   /** @inheritdoc */
   override def run[A](query: DBIO[A]): Future[A] =
     db.run(query)
```

Comment:
`db` is access without `synchronized` block. Also it may yield a `NullPointerException`.

Reply:
could probably use a double check where the first is non-synchronized and the the second one is synchroized, for performance reasons.

Reply:
I think it should be fine in our use case